### ISSUE TITLE
'ArrayIndexOutOfBoundsException: Coordinate out of bounds!' when resizing sketch and saving frame

### ISSUE THREAD

**Issue by clankill3r** [benfry/processing4#186]
When resizing the sketch really small and saving frames an ArrayIndexOutOfBoundsException is triggered.

```java
void setup() {
  surface.setResizable(true); 
}

void draw() {
  saveFrame();  
}
```

alpha 3
OSX 10.15.7

**Comment by benfry** [comment]
Marking this as lower priority… certainly shouldn't behave this way, but is there an actual use case for saving frames at 60 fps while resizing the window?

**Comment by benfry** [comment]
Fixed for 4.0 beta 5.
